tests/lean/eformat.lean

```diff
@@ -11,32 +11,32 @@ meta def trace_tagged_debug_locals : tactic unit := do
   cs ← cs.mmap infer_type,
   cs.mmap' trace_tagged_debug
 
-example (α : Type) (x : α) (h : 1 = 2) (f : ite ff true false) (g : x ≠ x) : true :=
+def example1 (α : Type) (x : α) (h : 1 = 2) (f : ite ff true false) (g : x ≠ x) : true :=
```

Comment:
Is there a `ite`/`dite`-related reason why these were changed to `def`s?

Reply:
It is an unrelated fix. The test compares against an expected output, but the order of the traces was nondeterministic as written.

Reply:
The *order* should always be deterministic (the tests run with `-j0`).  Are you maybe referring to some automatically generated names?

Reply:
I was typing a clarification :) I was debugging the test manually (it broke for `ite`-related reasons), and the nondeterminism was annoying. I realized after writing my previous comment that probably many tests rely on each test being run with a single thread, in which case I should probably revert this change.

Reply:
Pushed a revert.